modules/gridmap/grid_map_editor_plugin.cpp

```diff
@@ -524,8 +527,8 @@ void GridMapEditor::_set_clipboard_data() {
 
 				ClipboardItem item;
 				item.cell_item = itm;
-				item.grid_offset = Vector3(i, j, k) - selection.begin;
-				item.orientation = node->get_cell_item_orientation(i, j, k);
+				item.grid_offset = Vector3(ijk) - selection.begin;
```

Comment:
At first I thought that's some kind of C++ syntax which allows to group arguments. :eyes:

Reply:
I can see how this can be confusing from the diff, but I think it's clear when reading the code as a whole.

Do you have an alternative name suggestion? I just called it "ijk" because it can easily be understood as "[the vector containing] i, j, and k".

Reply:
In the context of this, perhaps `selected` or `pos` would be better IMO, unless it shadows other variables.

Reply:
Yeah I'd suggest `pos` too.

Reply:
Yeah, coming back to it I think `selected` makes a lot more sense, since these are all in a selection.

We should avoid `pos` since many instances of `pos` were switched to `position` in Godot 3.0, I think we should finish this work by converting the remaining instances of `pos` to `position` in Godot 4.0.